src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts

```diff
@@ -91,4 +85,12 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
       this.dtTrigger.next();
     }
   }
+
+  isLinkExternal(uri: string): WindowProxy | string {
+    return this.isValidUri(uri) ? window.open(uri, '_blank') : '';
+  }
+
+  isValidUri(uri: string) {
```

Comment:
I think all URIs are valid(at least by now, because we are not validating when the user types them). What we are checking is if the URI is a URL. A better name could be `isURL`

ref: https://danielmiessler.com/study/difference-between-uri-url/